spec/queries/keywords_query_spec.rb

```diff
@@ -18,7 +18,7 @@
 
       result = described_class.new(user).call
 
-      expect(result.count).to eq(3)
+      expect(result.to_a.size).to eq(3)
```

Comment:
Does it work without converting to an array? `result.size`

Reply:
Thanks, your comment opened my curiosity about "if the same type has both count/size, then what is the difference?!). 
[This post](https://stackoverflow.com/questions/6083219/activerecord-size-vs-count) let me with an open question: Do you have any preference between `length` and `size`? Length seems more specific to this need, while size more generic?
I'd be happy to hear your advises @junan  & @ankitkalia1195  :) 
Thanks!

Reply:
(both length and size are fully working in this example by the way~ )

Reply:
From the post
> if you already load all entries, say User.all, then you should use length to avoid another db query

As we loaded all the all entries/records(in `result ` ), it seems `length` will be good as it will avoid db query in this case

Reply:
Agreed. Fixed ;-)